change (humanoid): ban negative scale for humanoid bones (#360)
vrm-c/vrm-specification: specification/VRMC_vrm-1.0-beta/humanoid.md

<!-- END doctoc generated TOC please keep comment here to allow auto update -->

## Forbidden Transforms

> For transforms of humanoid bones, it MUST NOT have scales that have components of zero or negative.

Author: ```suggestion
For transforms of humanoid bones, scale components MUST have positive values (zero is not permitted).
```
